site/src/App/routes/gallery/Gallery.tsx

```diff
@@ -119,16 +129,21 @@ export const galleryIcons: typeof galleryComponents = Object.keys(icons).map(
   },
 );
 
-type SetName = 'components' | 'icons';
+type SetName = 'components' | 'layout' | 'icons';
 const getRowsFor = memoize((type: SetName) => {
-  const items = type === 'components' ? galleryComponents : galleryIcons;
+  const componentTypeMap = {
+    components: galleryContentComponents,
+    icons: galleryIcons,
+    layout: galleryLayoutComponents,
+  };
+
+  const items = componentTypeMap[type];
 
   const ratio = Math.max(
-    (window.innerWidth / window.innerHeight) *
-      (type === 'components' ? 0.65 : 0.5),
+    (window.innerWidth / window.innerHeight) * (type === 'icons' ? 0.5 : 0.65),
     1,
   );
-  const rowLength = Math.floor(Math.sqrt(items.length) * ratio);
+  const rowLength = Math.ceil(Math.sqrt(items.length) * ratio);
```

Comment:
`ceil` seems to work better to for all sections. Using `floor` here would have `rowLength` a bit too short for the Layout section